client.py

```diff
@@ -44,6 +50,24 @@ def __recv(self,sock,delim='\n',recv_buffer=4096):
                     raise Exception('Server Error: '+msg['response']+'\n'+msg['traceback'])
                 else:
                     return msg['response']
+    
+    def __handshake(self,handshake):
```

Comment:
is this used in the `com` method? It could be if you conditionally leave the socket open based on perhaps another input arg.